README.md

```diff
@@ -1,2 +1,57 @@
 # TWOFCS-2026
 Files and information related to the TWOFCS 2026 conference
+
+## Important Details
+Some important details about the conference at a glance:
+
+### Dates
+| Date                       | Deadline               |
+|----------------------------|------------------------|
+| Sep - Nov 30, 2025         | Abstract Submission    |
+| Nov 1 2025 - Jan 31, 2026  | Early Registration     |
+| Feb 1 - Apr 14, 2026       | Regular Registration   |
+| Apr 15 - Apr 17, 2026      | TWOFCS 2026            |
+| Date to be Announced       | Full Paper Close       |
+
+### Fees
+| Item                       | Fee                    |
+|----------------------------|------------------------|
+| General (Early)            | ¥15,000                |
+| General (Regular)          | ¥20,000                |
+| Student                    | Free                   |
+| Full Paper                 | Coming Soon            |
+
+### Code Showcase Registration
+To register for the Code Showcase, please fill out the form [here](https://peatix.com/us/event/4527366). The form is not available before September 1st, 2025.
+
+### Abstract Submission
+#### General Information:
+ - Abstract should be no longer than 5 pages
+ - Presentations should be no longer than 15 minutes
+ - Presentations are given 5 minutes of Q&A
+ - Technical talks will be grouped into sessions
+ - Abstract Template Available [here](https://science-tokyo.box.com/s/ukdu5xm1om8jetsmptswxds7bzcan6y3).
+
+#### Potential Topics Include (but are not limited to):
+ - Fundamentals of nuclear fuel cycle systems. Modeling. Development of analysis methods, uncertainty assessment, etc.
+ - Code development for nuclear fuel cycle simulators
+ - Analysis of future nuclear energy utilization and nuclear fuel cycles
+ - Policies, economic feasibility, social acceptability, nuclear non-proliferation safeguards, etc., for future nuclear energy utilization
+
+## Overview of the Conference
+The **7th Technical Workshop on Fuel Cycle Simulation (TWoFCS) 2026** will be held jointly with **NuMBAC 2026** in Tokyo, Japan, from **April 15–17, 2026**. This international workshop gathers experts to discuss nuclear and fuel cycle strategies using numerical analysis, integrating both technical and social science perspectives under the theme:
```

Comment:
I suggest moving the paragraph to the very top and removing this header

Reply:
Maybe also add a direct link to the website here. And a disclaimer that the website will have the most up-to-date information.